Description: The upstream examples don't use the Debian installed versions of
 files and the Debian packaging doesn't work with the provided relative
 paths. This patch makes the examples use the system-wide files. (Most required
 changes are done via sed commands in debian/rules).
Author: Paul Gevers <elbrus@debian.org>
Forwarded: not-needed

--- a/demos/effect/removeClass.html
+++ b/demos/effect/removeClass.html
@@ -5,8 +5,8 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>jQuery UI Effects - removeClass Demo</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/effect.js"></script>
+	<script src="/usr/share/javascript/jquery/jquery.js"></script>
+	<script src="/usr/share/javascript/jquery-ui/ui/effect.js"></script>
 	<link rel="stylesheet" href="../demos.css">
 	<style>
 	.toggler { width: 500px; height: 200px; position: relative; }
--- a/demos/slider/multiple-vertical.html
+++ b/demos/slider/multiple-vertical.html
@@ -5,11 +5,11 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>jQuery UI Slider - Multiple sliders</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/mouse.js"></script>
-	<script src="../../ui/slider.js"></script>
+	<script src="/usr/share/javascript/jquery/jquery.js"></script>
+	<script src="/usr/share/javascript/jquery-ui/ui/core.js"></script>
+	<script src="/usr/share/javascript/jquery-ui/ui/widget.js"></script>
+	<script src="/usr/share/javascript/jquery-ui/ui/mouse.js"></script>
+	<script src="/usr/share/javascript/jquery-ui/ui/slider.js"></script>
 	<link rel="stylesheet" href="../demos.css">
 	<style>
 	#eq > span {
--- a/demos/tooltip/custom-style.html
+++ b/demos/tooltip/custom-style.html
@@ -4,7 +4,7 @@
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>jQuery UI Tooltip - Custom Styling</title>
-	<link rel="stylesheet" href="../../themes/base/tooltip.css">
+	<link rel="stylesheet" href="/usr/share/javascript/jquery-ui/css/base/tooltip.css">
 	<link rel="stylesheet" href="../demos.css">
 	<script src="../../external/requirejs/require.js"></script>
 	<script src="../bootstrap.js">
--- a/demos/bootstrap.js
+++ b/demos/bootstrap.js
@@ -79,15 +79,10 @@
 document.documentElement.className = "demo-loading";
 
 require.config( {
-	baseUrl: window.location.pathname.indexOf( "demos/" ) !== -1 ? "../../ui" : "../../../ui",
+	baseUrl: "/usr/share/javascript/jquery-ui/ui",
 	paths: {
-		jquery: "../external/jquery/jquery",
-		external: "../external/"
+		jquery: "/usr/share/javascript/jquery/jquery",
 	},
-	shim: {
-		"external/globalize/globalize.culture.de-DE": [ "external/globalize/globalize" ],
-		"external/globalize/globalize.culture.ja-JP": [ "external/globalize/globalize" ]
-	}
 } );
 
 // Replace effects all shortcut modules with all the effects modules
--- a/demos/spinner/default.html
+++ b/demos/spinner/default.html
@@ -7,7 +7,7 @@
 	<link rel="stylesheet" href="../../themes/base/all.css">
 	<link rel="stylesheet" href="../demos.css">
 	<script src="../../external/requirejs/require.js"></script>
-	<script src="../bootstrap.js" data-modules="external/jquery-mousewheel/jquery.mousewheel">
+	<script src="../bootstrap.js" data-modules="/usr/share/javascript/jquery-mousewheel/jquery.mousewheel.js">
 		var spinner = $( "#spinner" ).spinner();
 
 		$( "#disable" ).on( "click", function() {
--- a/demos/spinner/latlong.html
+++ b/demos/spinner/latlong.html
@@ -8,7 +8,7 @@
 	<link rel="stylesheet" href="../demos.css">
 	<script src="http://maps.google.com/maps/api/js?sensor=false"></script>
 	<script src="../../external/requirejs/require.js"></script>
-	<script src="../bootstrap.js" data-modules="external/jquery-mousewheel/jquery.mousewheel">
+	<script src="../bootstrap.js" data-modules="/usr/share/javascript/jquery-mousewheel/jquery.mousewheel.js">
 		function latlong() {
 			return new google.maps.LatLng( $("#lat").val(), $("#lng").val() );
 		}
--- a/demos/spinner/overflow.html
+++ b/demos/spinner/overflow.html
@@ -7,7 +7,7 @@
 	<link rel="stylesheet" href="../../themes/base/all.css">
 	<link rel="stylesheet" href="../demos.css">
 	<script src="../../external/requirejs/require.js"></script>
-	<script src="../bootstrap.js" data-modules="external/jquery-mousewheel/jquery.mousewheel">
+	<script src="../bootstrap.js" data-modules="/usr/share/javascript/jquery-mousewheel/jquery.mousewheel.js">
 		$( "#spinner" ).spinner({
 			spin: function( event, ui ) {
 				if ( ui.value > 10 ) {
